mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentDAOTest.java

```diff
@@ -51,15 +51,15 @@
 
     @Before
     public void setUp() throws Exception {
-        cassandra = CassandraCluster.create(new CassandraAttachmentModule(),
-            cassandraServer.getIp(), cassandraServer.getBindingPort());
+        cassandra = CassandraCluster.create(CassandraAttachmentModule.CASSANDRA_ATTACHMENT_TABLE,
```

Comment:
Let's discuss that a bit more, maybe that change (module breaking) i no more very significant:
 - Performance gain was little
 - Smaller modules allow less table creation, which have an impact when you create tables on each test. Less when you create tables once per running class
 - Grouping classes together to avoid launching too much docker requires to create tables for all the tested class at once. 

We could keep the Staged builder, and drop the splits...

Reply:
If gains are small, I would prefer dropping the splits, yes

Reply:
Ok